autoria.py

```diff
@@ -399,11 +399,26 @@ def select_item(item_to_select: str, items_list: dict) -> int:
             needed item (category, bodystyle, mark etc.) identifyer.
         """
         if item_to_select is not None and items_list is not None:
+            result = None
             for item in items_list:
-                if fnmatch(item['name'], '*' + item_to_select + '*'):
-                    return item['value']
-        else:
-            return None
+                if fnmatch(item['name'], item_to_select):
+                    result = item['value']
+                    print(result)
+                    return result
+            if result is None:
```

Comment:
because if the match is found in the first loop, it would be returned, if not - the interpreter would go further, to the next loop, the same logic happens there, if there's still no match, it would go to the third loop..
Removed ifs :)